src/labels/label_line.js

```diff
@@ -250,12 +251,15 @@ export default class LabelLine {
 
         switch (this.placement) {
             case PLACEMENT.CORNER:
-                position = segment[1].slice();
+                position = [
+                    Utils.roundToNearestPixel(segment[1][0]),
```

Comment:
Or I guess since the tile units are 4096 x 4096, and screen space is 256 x 256, we could round to the nearest 1/16th? (floating point error in JS is another problem to consider here)